images/x86_64/CentOS_7/shippable.yml

```diff
@@ -1265,6 +1265,55 @@ jobs:
       script:
         - *final_on_failure_template
 
+  - name: v6114_c7_x8664_gce_img
+    type: runSh
+    dependencyMode: strict
+    triggerMode: parallel
+    steps:
+      - IN: node_repo_file_tag
+      - IN: cexec_repo_file_tag
+      - IN: c7repLib_x8664_tag
+      - IN: reqKick_repo_file_tag
+      - IN: u16reqProc_dd_tag
+      - IN: u16reqProc_dd_img
+        switch: off
+      - IN: buildami_repo
+        switch: off
+      - IN: prod_release
+        switch: off
+      - IN: ship_bits_gcp
+        switch: off
+      - IN: patch_c7_x8664_gce_img
+        switch: off
+      - TASK:
+          name: build_gce_image
+          runtime:
+            options:
+              env:
+                - BASE_IMG_RES: "patch_c7_x8664_gce_img"
+                - DOCKER_VER: 18.03
+                - NODE_RES: "node_repo_file_tag"
+                - REQPROC_RES: "u16reqProc_dd_tag"
+                - REQPROC_IMG_RES: "u16reqProc_dd_img"
+                - REPORTS_RES: "c7repLib_x8664_tag"
+                - REQKICK_RES: "reqKick_repo_file_tag"
+                - CEXEC_RES: "cexec_repo_file_tag"
+                - MACHINE_TYPE: "n1-standard-8"
+                - REGION: "us-east1"
+                - ZONE: "us-east1-b"
+                - PROJECT_ID: "ship-bits"
+                - SERVICE_ACCOUNT_JSON: "gcp_key.json"
+                - SSH_USER: "centos"
+                - IMG_VER: "v6.11.4"
+          script:
+            - *onetime_final
```

Comment:
I don't think we need to use `onetime_final` any more. Please confirm with @Bharath92.

Reply:
we need to use `onetime_final` so that the `SOURCE_IMAGE_FAMILY` gets populated from the patch job

Reply:
Ok, I remember why now. We don't yet have a `shipctl` helper to read current job state, which is needed to consolidate this.